### PR TITLE
add back infinite duality gap

### PR DESCRIPTION
**Pull request by chriscoey**

### CODE REVIEW

File: examples/dualitygap.jl [hunk 1]
```diff
@@ -0,0 +1,44 @@
+
+# Example of polyhedral OA failure due to infinite duality gap from "Polyhedral approximation in mixed-integer convex optimization - Lubin et al 2016"
```
Comment (mlubin) [inline examples/dualitygap.jl]: Not a great example showing off Pajarito on a problem that we can't solve

Comment (chriscoey) [inline examples/dualitygap.jl]: people gotta know the limitations

Comment (chriscoey) [inline examples/dualitygap.jl]: i'll make a note that it should fail

Comment (chriscoey) [inline examples/dualitygap.jl]: also for now these are mostly just to make it easy for us to run problems and observe output to debug

**Pull request merged**